packages/core/phase2/exceptions/HO200103.ts

```diff
@@ -1,9 +1,32 @@
+import ExceptionCode from "bichard7-next-data-latest/dist/types/ExceptionCode"
+import errorPaths from "../../lib/exceptions/errorPaths"
 import type { AnnotatedHearingOutcome } from "../../types/AnnotatedHearingOutcome"
 import type Exception from "../../types/Exception"
 import type { ExceptionGenerator } from "../../types/ExceptionGenerator"
+import ResultClass from "../../types/ResultClass"
 
-const generator: ExceptionGenerator = (_aho: AnnotatedHearingOutcome, _options): Exception[] => {
-  return []
+const generator: ExceptionGenerator = (aho: AnnotatedHearingOutcome): Exception[] => {
+  const exceptions: Exception[] = []
+  const offences = aho.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.Offence
+
+  offences.forEach((offence, offenceIndex) => {
```

Comment:
We also need to check whether both offence and result are recordable, or use the `checkResultClassExceptions` function.